pkg/virt-api/webhooks/fuzz/fuzz_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package fuzz_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestFuzz(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Fuzz Suite")
```

Comment:
Yeah good point, interesting read:
https://github.com/onsi/ginkgo/issues/714#issuecomment-678624224

I think fuzz_test specifically works because it is not ginkgo?

Reply:
Maybe the fuzz one indeed, but the others are ginkgo.

Reply:
I guess they are all run because they had the test suite setup func in `_test.go`
But consider new PR submissions:
- We have no CI gate that would catch a suite_file missing
- We don't have anything that would catch KubeVirtTestSuiteSetup missing
So PRs could get merged with their unit tests not being ran

Reply:
Well, if an author adds a test that does not run, it means he did not write the test well.
If a reviewer and the approval missed it, bad for them as well.

I do not understand how this has to do with making sure someone will do the job correctly in the future.

I think the PR needs to first fix the problem of not running some tests and suggest in a follow up PR a way to assure this will not happen again (not that I understand how at the moment).
Lets not mix things up, it will be hard to understand the changes later.

Reply:
> suggest in a follow up PR a way to assure this will not happen again

I believe this is the intention of running the script I am fixing. Today, it is just broken due to not having the ginkgo binary